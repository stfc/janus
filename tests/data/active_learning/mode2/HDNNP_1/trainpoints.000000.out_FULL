################################################################################
# Energy comparison.
################################################################################
# Col  Name  Description
################################################################################
# 1    index Structure index.
# 2    Eref  Reference potential energy per atom (training units).
# 3    Ennp  NNP potential energy per atom (training units).
################################################################################
#        1                2                3
#    index             Eref             Ennp
############################################
        25   4.35157429E+03  -9.99730691E-01
        36   4.35157429E+03  -1.13181304E+00
        64   4.35157429E+03  -1.83251877E+00
        82   4.35157429E+03  -1.54727742E+00
       119   4.35157429E+03  -3.35993703E-01
       160   4.35157429E+03  -1.40596323E+00
       174   4.35157429E+03  -9.19460929E-01
       181   4.35157429E+03  -8.52765944E-01
        26   4.35157429E+03  -1.21549578E+00
        85   4.35157429E+03  -1.42463655E+00
       101   4.35157429E+03  -1.27051232E+00
       122   4.35157429E+03  -9.05967961E-01
       144   4.35157429E+03  -7.94258683E-01
       146   4.35157429E+03  -9.04893470E-01
       170   4.35157429E+03  -8.09356951E-01
       175   4.35157429E+03  -1.48074553E+00
        11   4.35157429E+03  -3.51031308E-01
        19   4.35157429E+03  -8.89958302E-01
        61   4.35157429E+03  -1.98991767E+00
        72   4.35157429E+03  -1.73955811E+00
        77   4.35157429E+03  -1.77424905E+00
        79   4.35157429E+03  -1.78727602E+00
       104   4.35157429E+03  -1.74184855E+00
       108   4.35157429E+03  -1.75055176E+00
        24   4.35157429E+03  -1.27423850E+00
        49   4.35157429E+03  -2.02693253E+00
        68   4.35157429E+03  -1.82595539E+00
        97   4.35157429E+03  -1.52592578E+00
       114   4.35157429E+03  -4.46326570E-01
       162   4.35157429E+03  -4.86554528E-01
       165   4.35157429E+03  -1.27447514E+00
       172   4.35157429E+03  -7.09834843E-01
         5   4.35157429E+03  -2.05106211E-01
        10   4.35157429E+03  -8.31004346E-02
        47   4.35157429E+03  -1.62774341E+00
        53   4.35157429E+03  -1.16958105E+00
        84   4.35157429E+03  -1.18620132E+00
       134   4.35157429E+03  -1.78994282E+00
       157   4.35157429E+03  -1.37735844E+00
       159   4.35157429E+03  -1.18826663E+00
        30   4.35157429E+03  -1.34253214E+00
        40   4.35157429E+03  -1.01518079E+00
        69   4.35157429E+03  -1.32596114E+00
        70   4.35157429E+03  -1.87730246E+00
        83   4.35157429E+03  -1.66589983E+00
        89   4.35157429E+03  -1.00452715E+00
       100   4.35157429E+03  -1.77530361E+00
       152   4.35157429E+03  -9.83555171E-01
         8   4.35157429E+03  -6.04898425E-01
        17   4.35157429E+03  -1.31241299E+00
        66   4.35157429E+03  -1.50346762E+00
        86   4.35157429E+03  -1.08221823E+00
        96   4.35157429E+03  -1.54734182E+00
       103   4.35157429E+03  -1.47534791E+00
       113   4.35157429E+03  -9.78408766E-01
       118   4.35157429E+03  -4.55674082E-01
        63   4.35157429E+03  -1.92776081E+00
       111   4.35157429E+03  -8.51876836E-01
       130   4.35157429E+03  -6.81541717E-01
       154   4.35157429E+03  -9.62452020E-01
       186   4.35157429E+03  -8.63188290E-01
       189   4.35157429E+03  -4.07889665E-01
       190   4.35157429E+03  -6.43305657E-01
       191   4.35157429E+03  -1.06546913E+00
        14   4.35157429E+03  -1.88760328E+00
        16   4.35157429E+03  -1.61019789E+00
        31   4.35157429E+03  -1.72438102E+00
        34   4.35157429E+03  -1.97062822E+00
        44   4.35157429E+03  -1.82721207E+00
        51   4.35157429E+03  -1.27580552E+00
        52   4.35157429E+03  -1.88835778E+00
        54   4.35157429E+03  -1.81169974E+00
        38   4.35157429E+03  -1.66643695E+00
        58   4.35157429E+03  -1.27860504E+00
        91   4.35157429E+03  -1.14056056E+00
        93   4.35157429E+03  -7.57456240E-01
       133   4.35157429E+03  -1.68934267E+00
       140   4.35157429E+03  -1.04908615E+00
       145   4.35157429E+03  -1.24712822E+00
       151   4.35157429E+03  -1.11808861E+00
         1   4.35157429E+03   1.00148623E+00
        37   4.35157429E+03  -2.01613670E+00
        73   4.35157429E+03  -1.25390774E+00
        98   4.35157429E+03  -1.54106277E+00
       126   4.35157429E+03  -9.70135420E-01
       161   4.35157429E+03  -1.44489811E+00
       184   4.35157429E+03  -7.23768647E-01
       188   4.35157429E+03  -1.02205112E+00
        42   4.35157429E+03  -1.57828813E+00
        50   4.35157429E+03  -1.32762485E+00
        75   4.35157429E+03  -1.76509603E+00
       102   4.35157429E+03  -1.52675798E+00
       109   4.35157429E+03  -1.11633586E+00
       139   4.35157429E+03  -8.92981309E-01
       173   4.35157429E+03  -9.17011604E-01
       178   4.35157429E+03  -8.72646768E-01
         3   4.35157429E+03  -5.36051689E-02
        23   4.35157429E+03  -1.13353279E+00
        62   4.35157429E+03  -1.45362692E+00
        80   4.35157429E+03  -1.21043566E+00
       105   4.35157429E+03  -1.34005891E+00
       123   4.35157429E+03  -7.40812045E-01
       148   4.35157429E+03  -1.28794879E+00
       156   4.35157429E+03  -1.14874285E+00
         0   4.35157429E+03   6.94880925E-01
        18   4.35157429E+03  -1.50165579E+00
        21   4.35157429E+03  -9.94906649E-01
       107   4.35157429E+03  -1.55578124E+00
       121   4.35157429E+03  -2.57046823E-01
       153   4.35157429E+03  -1.13078020E+00
       182   4.35157429E+03  -1.13289975E+00
       185   4.35157429E+03  -8.37354720E-01
        29   4.35157429E+03  -1.16329975E+00
        46   4.35157429E+03  -2.01415320E+00
        99   4.35157429E+03  -1.07586943E+00
       127   4.35157429E+03  -7.17815521E-01
       158   4.35157429E+03  -1.07459022E+00
       169   4.35157429E+03  -2.48497990E-01
       171   4.35157429E+03  -6.26776783E-01
       176   4.35157429E+03  -8.88736555E-01
         6   4.35157429E+03   1.79178164E-01
        27   4.35157429E+03  -8.86073351E-01
        33   4.35157429E+03  -1.09122414E+00
       110   4.35157429E+03  -1.06862570E+00
       129   4.35157429E+03  -1.42399595E+00
       163   4.35157429E+03  -8.76149760E-01
       179   4.35157429E+03  -1.30298705E+00
       180   4.35157429E+03  -1.41880072E+00
         2   4.35157429E+03   1.09911995E-01
        20   4.35157429E+03  -1.04290198E+00
        71   4.35157429E+03  -1.38458232E+00
        74   4.35157429E+03  -1.26165485E+00
        92   4.35157429E+03  -1.34723190E+00
       125   4.35157429E+03  -8.32524880E-01
       128   4.35157429E+03  -6.00650013E-01
       155   4.35157429E+03  -9.90215666E-01
         7   4.35157429E+03  -2.78350161E-01
        94   4.35157429E+03  -1.94323844E+00
       131   4.35157429E+03  -1.31848801E+00
       150   4.35157429E+03  -1.26758459E+00
       164   4.35157429E+03  -1.12924512E+00
       166   4.35157429E+03  -1.34984606E+00
       168   4.35157429E+03  -9.95550964E-01
       177   4.35157429E+03  -1.01453840E+00
         9   4.35157429E+03  -8.83695527E-01
        32   4.35157429E+03  -1.80430991E+00
        59   4.35157429E+03  -1.40184425E+00
       112   4.35157429E+03  -6.07909642E-01
       135   4.35157429E+03  -8.50484972E-01
       138   4.35157429E+03  -1.57543351E+00
       141   4.35157429E+03  -6.09850640E-01
       167   4.35157429E+03  -1.33576742E+00
        22   4.35157429E+03  -9.47639155E-01
        55   4.35157429E+03  -1.95155868E+00
        57   4.35157429E+03  -1.67617848E+00
        60   4.35157429E+03  -1.92816171E+00
        90   4.35157429E+03  -1.35277685E+00
        95   4.35157429E+03  -1.26012699E+00
       106   4.35157429E+03  -1.54188559E+00
       115   4.35157429E+03  -2.82097258E-01
        12   4.35157429E+03  -4.14426499E-01
        13   4.35157429E+03  -1.09953100E+00
        28   4.35157429E+03  -1.88411202E+00
        41   4.35157429E+03  -1.32554054E+00
        56   4.35157429E+03  -1.24807593E+00
        76   4.35157429E+03  -1.57889231E+00
        88   4.35157429E+03  -1.15707395E+00
       132   4.35157429E+03  -7.71132549E-01
        48   4.35157429E+03  -1.75291059E+00
        67   4.35157429E+03  -1.67320052E+00
       120   4.35157429E+03  -7.85221175E-01
       136   4.35157429E+03  -8.03532373E-01
       143   4.35157429E+03  -1.44478298E+00
       149   4.35157429E+03  -8.91199318E-01
       183   4.35157429E+03  -1.40438362E+00
       187   4.35157429E+03  -8.99337622E-01
        39   4.35157429E+03  -1.83482675E+00
        43   4.35157429E+03  -1.51222237E+00
        45   4.35157429E+03  -2.02803228E+00
        65   4.35157429E+03  -1.82583012E+00
       116   4.35157429E+03  -1.45923308E+00
       124   4.35157429E+03  -1.09593661E+00
       142   4.35157429E+03  -1.06670559E+00
       147   4.35157429E+03  -8.62139414E-01
         4   4.35157429E+03   6.06481014E-01
        15   4.35157429E+03  -7.41564631E-01
        35   4.35157429E+03  -2.00018212E+00
        78   4.35157429E+03  -1.34740207E+00
        81   4.35157429E+03  -1.26069290E+00
        87   4.35157429E+03  -9.74831381E-01
       117   4.35157429E+03  -1.27150219E+00
       137   4.35157429E+03  -1.43480021E+00
