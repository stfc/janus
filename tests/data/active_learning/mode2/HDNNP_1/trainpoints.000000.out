################################################################################
# Energy comparison.
################################################################################
# Col  Name  Description
################################################################################
# 1    index Structure index.
# 2    Eref  Reference potential energy per atom (training units).
# 3    Ennp  NNP potential energy per atom (training units).
################################################################################
#        1                2                3
#    index             Eref             Ennp
############################################
         0   4.35157429E+03  -9.99730691E-01
